src/plugin/range/index.js

```diff
@@ -0,0 +1,65 @@
+
+export class DayjsRange {
+  constructor(start, end) {
+    let s = start
+    let e = end
+
+    if (s > e) {
+      s = end
+      e = start
+    }
+
+    this.start = s
+    this.end = e
+  }
+
+  isRange() {
+    return this.start.isValid() && this.end.isValid() && this instanceof DayjsRange
```

Comment:
what's the point of having this as a class method that checks whether the current instance is a range? The answer will always be yes. Also I don't think isRange should report false if one of the dates is invalid, it's still a range. Perhaps an isValid() method would be more useful.

Reply:
understood this comment. i will change to `isValidRange`.
and validations `this.start`, `this.end`

i guess "end date is after the start state." has already been confirmed in 'constructor'.
need more?

Reply:
no you're right, I missed the constructor bit!

Reply:
Reflected you are feedback. 
Try check again please :pray: